src/app/modules/main/chat_section/model.nim

```diff
@@ -314,6 +318,17 @@ QtObject:
       if self.items[i].subItems.updateNotificationsForItemById(id, hasUnreadMessages, notificationsCount):
         return
 
+  proc updateLastMessageTimestampForItemOrSubItemById*(self: Model, id: string, lastMessageTimestamp: int) =
+    for i in 0 ..< self.items.len:
+      if(self.items[i].id == id):
+        let index = self.createIndex(i, 0, nil)
+        self.items[i].BaseItem.lastMessageTimestamp = lastMessageTimestamp
+        self.dataChanged(index, index, @[ModelRole.LastMessageTimestamp.int])
+        return
+
+      if self.items[i].subItems.updateLastMessageTimestampForItemById(id, lastMessageTimestamp):
```

Comment:
@osmaczko since this ticket says that we should move the most recent created chat or the chat with most recent message to top of the chat list, but only for the `Chat` section, not for any of community sections, this part is not needed. We don't need to check `subItems` as it refers only to the chats which are part of categories which are available only for communities. 

Feel free to remove this part and rename `updateLastMessageTimestampForItemOrSubItemById` appropriately cause it won't update sub model any more.